### PR TITLE
0.2/790 sort families on group collection sheet

### PR DESCRIPTION
**Pull request by charlesstrange2**

### CODE REVIEW

File: app/Family.php [hunk 1]
```diff
@@ -279,4 +279,15 @@ public function initialCentre()
     {
         return $this->belongsTo('App\Centre', 'initial_centre_id');
     }
+
+    public function scopeWithPrimaryCarer($query)
+    {
+        $subQuery = \DB::table('carers')
```
Comment (katjam) [inline app/Family.php:285]: lol the old subQuery trick!

File: app/Family.php [hunk 2]
```diff
@@ -279,4 +279,15 @@ public function initialCentre()
...
+            ->select('name')
+            ->whereRaw('family_id = families.id')
+            ->oldest()
```
Comment (katjam) [inline app/Family.php]: It's always the first input? And can not be amended?

File: app/Registration.php [hunk 3]
```diff
@@ -110,7 +110,10 @@ public function getReminderReasons()
     public function scopeWithFullFamily($query)
     {
         return $query->with([
-            'family',
+            // This may not be efficient, but it is convenient for odering when required.
```
Comment (katjam) [inline app/Registration.php]: ordering typo

**Review by katjam — APPROVED** [review]
Yep. I like it as long as oldest() does always get the right one.

File: app/Family.php (hunk 1 above)
Comment (charlesstrange2) [reply app/Family.php:285]: trying new things!

File: app/Family.php (hunk 2 above)
Comment (charlesstrange2) [reply app/Family.php]: Will check...

**Pull request merged**